### PR TITLE
DOCS | trying to fix broken version links

### PR DESCRIPTION
**Pull request by immber**
V4 Documentation link is currently broken on https://docs.coralproject.net/talk 

This update should fix the version links on the sidebar!

### CODE REVIEW

**Review by kgardnr — APPROVED** [review]

**Review by kgardnr — COMMENTED** [review]
Actually let's just wait until tomorrow to merge this

**Comment by wyattjoh** [comment]
Fixed by turning off post processing.

**Pull request closed without merging**
